docs/Users_Guide/wrappers.rst

```diff
@@ -867,12 +867,13 @@ Description
 -----------
 
 The ExtractTiles wrapper is used to regrid and extract subregions from
-paired tropical cyclone tracks that are created by the tc_pairs_wrapper.
+paired tropical cyclone tracks. The tracks are created from output from
+TCStat or MTD.
```

Comment:
Should we also add something here specifying it only reads cluster object pairs?

Reply:
Sure, feel free to make any adjustments to the documentation to make things more clear.